ui/yarn.lock

```diff
@@ -6167,19 +6167,10 @@ loader-runner@^4.2.0:
   resolved "https://registry.yarnpkg.com/loader-runner/-/loader-runner-4.2.0.tgz#d7022380d66d14c5fb1d496b89864ebcfd478384"
   integrity sha512-92+huvxMvYlMzMt0iIOukcwYBFpkYJdpl2xsZ7LrlayO7E8SOv+JJUEK17B/dJIHAOLMfh2dZZ/Y18WgmGtYNw==
 
-loader-utils@^1.4.0:
```

Comment:
I deleted this section manually, and it was **not** regenerated because there is now nothing that depends on loader-utils 1.x

Reply:
actually I think it would have deleted this without my action, for that reason